get_started/concepts.rst

```diff
@@ -1,26 +1,20 @@
 .. _citus_concepts:
 
-
-Concepts
```

Comment:
I would still like to keep the Concepts part. In the left TOC, seeing Architecture --> Nodes seems unclear. We could have just 2 top-level items there for now "Concepts" and the MX architecture.

Reply:
Restored it.